### ISSUE TITLE
Delocalization Subgraph Section

### ISSUE THREAD

**Issue by rapodaca** [rapodaca/dialect#8]
- VB model makes graphs unequal even though they should be treated as equal
  - DIME
- the DS solution
  - encodes alternating single-double bond pattern as a perfect matching
    - maximum, maximal, perfect matching
- a node-induced subgraph over VB graph
  - unlabeled
  - possibly empty
  - encodes and therefore guarantees a perfect matching
    - corollary: every atom will be assigned a double bond
  - only node membership is specified
    - edge membership is deduced
- composition
  - atoms
    - C, N, O, P, S, unknown
  - selection
    - adding an atom to the DS
    - "selected atom", "selected atoms"
    - "induced bond", "induced bonds"
- bonds
  - by definition (node-induced) both terminals are members
- literature refers to "aromaticity"
  - not using that term because of its (imprecise) meaning in chemistry
- algorithmic fill/empty
  - every molecule expressed with a DS can also be expressed without it
  - in other words, 1:1 translation
  - fill algorithm
  - empty algorithm
    - aka "kekulization"
- examples
  - invent a notation for node/edge membership (color?, weight?)
  - graphics only
- error states
  - no perfect matching possible
  - to re-iterate: DS is just another way to express single/double bond layout
